crypto/cryptoutil/cert.go

```diff
@@ -0,0 +1,31 @@
+package cryptoutil
+
+import (
+	"crypto/x509"
+	"encoding/pem"
+	"os"
+)
+
+// ReadCertificateFile reads a certificate PEM file.
```

Comment:
Do we plan to support DER format?

Reply:
PEM is a wrapper of DER cert. If it is a DER cert, just call `x509.ParseCertificate()` directly.